engine/runner/src/main/resources/META-INF/native-image/org/enso/runner/native-image.properties

```diff
@@ -1 +1,2 @@
-Args=--features=org.enso.runner.EnsoLibraryFeature
+Args = --features=org.enso.runner.EnsoLibraryFeature \
```

Comment:
- This is not good idea.
- Foreign support doesn't work on Mac Arm
- we have to run without such a flag at least in GraalVM24

Reply:
It seems that for now, it is best to wait for the GraalVM 25 release, scheduled for 2025-09-16, which will include support for ARM64 (https://github.com/oracle/graal/issues/8113#issuecomment-2776044177). Otherwise, we will need to somehow conditionally enable/disable `-H:+ForeignAPISupport` for certain architectures.

Reply:
After our 1:1 meeting:
- Your suggestion deals with allocating new off-heap memory and copying bytes into it
- It does not deal with `mmap`.
- We would need to implement native calls of `mmap` ourselves
- Which seems difficult
- Conclusion: Waiting for the GraalVM 25 release and just use `Arena` for `mmap`.

Reply:
> * Conclusion: Waiting for the GraalVM 25 release

- OK, blocked by #13918